src/lib/components/forms/form/form-components/text-area/text-area.component.spec.ts

```diff
@@ -0,0 +1,143 @@
+import { MatInputHarness } from '@angular/material/input/testing';
+import { MatFormFieldHarness } from '@angular/material/form-field/testing';
+import { By } from '@angular/platform-browser';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { TerraPlacementEnum } from '../../../../../helpers';
+import { MockTooltipDirective } from '../../../../../testing/mock-tooltip.directive';
+import { TextAreaComponent } from './text-area.component';
+
+// tslint:disable-next-line: max-function-line-count
+describe('TextAreaComponent', () => {
+    let component: TextAreaComponent;
+    let fixture: ComponentFixture<TextAreaComponent>;
+    let loader: HarnessLoader;
+    let input: MatInputHarness;
+    let host: TestElement;
+
+    const testString: string = 'foo';
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            declarations: [MockTooltipDirective, TextAreaComponent],
+            imports: [FormsModule, MatFormFieldModule, MatInputModule, NoopAnimationsModule]
+        }).compileComponents();
+        fixture = TestBed.createComponent(TextAreaComponent);
+        component = fixture.componentInstance;
+
+        loader = TestbedHarnessEnvironment.loader(fixture);
+        input = await loader.getHarness(MatInputHarness);
+        host = await input.host();
+
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+        expect(input).toBeTruthy();
+    });
+
+    it('should have type textarea', async () => {
+        expect(await input.getType()).toBe('textarea');
+        expect(await host.matchesSelector('textarea')).toBe(true);
+    });
+
+    it('should set name as mat-label', async () => {
+        component.name = testString;
+        fixture.detectChanges();
+        const formField: MatFormFieldHarness = await loader.getHarness(MatFormFieldHarness);
+
+        expect(await formField.getLabel()).toBe(testString);
+    });
+
+    it('should set required state according to #isRequired', async () => {
+        expect(await input.isRequired()).toBe(false);
+        component.isRequired = true;
+        fixture.detectChanges();
+        expect(await input.isRequired()).toBe(true);
+    });
+
+    it('should set disabled state according to #isDisabled', async () => {
+        expect(await input.isDisabled()).toBe(false);
+        component.isDisabled = true;
+        fixture.detectChanges();
+        expect(await input.isDisabled()).toBe(true);
+    });
+
+    describe('with tooltip', () => {
+        let tooltTip: MockTooltipDirective;
+
+        beforeEach(() => {
+            tooltTip = fixture.debugElement
+                .query(By.directive(MockTooltipDirective))
+                .injector.get(MockTooltipDirective);
+        });
+
+        it('should set tooltip placement according to #tooltipPlacement', () => {
+            expect(tooltTip.placement).toBe('top');
```

Comment:
Vllt auch hier die `TerraPlacementEnum` verwenden?
```suggestion
            expect(tooltTip.placement).toBe(TerraPlacementEnum.TOP);
```